blog/.vuepress/theme/partials/Header.vue

```diff
@@ -34,6 +34,26 @@ import { mapGetters } from "vuex";
 
 export default {
   props: ["header"],
+  data: function () {
+    return {
+      navActive: false
+    }
+  },
+  created () {
+    window.addEventListener('scroll', this.handleScroll);
+  },
+  destroyed () {
+    window.removeEventListener('scroll', this.handleScroll);
+  },
+  methods: {
+    handleScroll: function () {
```

Comment:
Since this is a code duplication I would suggest to implement the functionality as a vue `directive`. There is already a very good example with scrolling available: https://vuejs.org/v2/cookbook/creating-custom-scroll-directives.html

Reply:
I made a new directive directly in `enhanceApp.js`. Not sure if this is the right place.